src/styles/partials/components/_alert.scss

```diff
@@ -0,0 +1,115 @@
+.dg_alert {
+  background: $vibrant-blue;
+  padding: $default-padding;
+
+  .dg_alert__container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+
+    margin: auto;
+    max-width: 1400px;
+    width: 100%;
+  }
+
+  * {
+    color: $white;
+  }
+
+  a {
+    text-decoration: underline;
+
+    &:hover {
+      background: $white;
+      color: $vibrant-blue;
+    }
+  }
+
+  p {
+    font-size: $smallest-heading-font-size;
+    margin-bottom: 0;
+  }
+
+  p,
+  .dg_alert__status {
+    font-family: $default-heading-font;
+  }
+
+  .dg_alert__status {
+    font-weight: $font-bold;
+    text-transform: uppercase;
+  }
+
+  .dg_alert__icon {
+    font-size: $large-icon-size;
+    padding: 0 ($default-padding * 2) 0 $default-padding;
+  }
+
+  &.emergency {
```

Comment:
Looking at the [XD doc](https://xd.adobe.com/spec/ad47a25b-c580-4389-756b-04b95b9d8a00-db76/screen/98a9d726-b1a9-4d86-b2a2-27dec7beb09b/11-Alerts), should the Emergency and Default alerts be centered? If so, I would add `text-align: center;` to each.

Reply:
validated, and fixed. I had to adjust some of the flex stuff to handle this as well

Reply:
Text appears to now be centered on both the Emergency and Default alerts.